core/src/main/scala/com/raphtory/internals/context/RaphtoryContext.scala

```diff
@@ -1,48 +1,45 @@
 package com.raphtory.internals.context
 
 import cats.effect.IO
-import com.oblac.nomen.Nomen
+import cats.effect.Resource
 import com.raphtory.api.analysis.graphview.DeployedTemporalGraph
-import com.raphtory.internals.management._
+import com.raphtory.createName
+import com.raphtory.internals.communication.repositories.LocalTopicRepository
+import cats.effect.unsafe.implicits.global
+import com.raphtory.internals.management.Prometheus
+import com.raphtory.internals.management.QuerySender
+import com.raphtory.internals.management.Scheduler
+import com.raphtory.protocol.RaphtoryService
 import com.typesafe.config.Config
 import com.typesafe.scalalogging.Logger
 import org.slf4j.LoggerFactory
+import com.raphtory.internals.components.querymanager.Query
+
+class RaphtoryContext(serviceResource: Resource[IO, RaphtoryService[IO]], config: Config, local: Boolean = true) {
+  protected val logger: Logger = Logger(LoggerFactory.getLogger(this.getClass))
+
+  def newGraph(graphID: String = createName): DeployedTemporalGraph = {
+    val qs = for {
+      _          <- Prometheus[IO](config.getInt("raphtory.prometheus.metrics.port"))
+      service    <- serviceResource
+      topicRepo  <- LocalTopicRepository[IO](config, None)
+      querySender = new QuerySender(graphID, service, new Scheduler(), topicRepo, config, createName)
+      _          <- Resource.eval(IO(querySender.establishGraph()))
+    } yield querySender
+
+    val (querySender, shutdown) = qs.allocated.unsafeRunSync()
+    new DeployedTemporalGraph(Query(graphID = graphID), querySender, config, local, shutdown)
```

Comment:
I think we are not using the `local` parameter anymore. Maybe we can remove it.

Reply:
@miratepuffin Could you please confirm?

Reply:
yeah I believe that is the case, the only place I think it was being used was the different behaviour for local and remote destroy/closure which obviously all changes with this.
<img width="404" alt="image" src="https://user-images.githubusercontent.com/6665739/196155456-f5c5e5f8-23a9-41c7-8bb9-93d527ad1d30.png">